Implemented API

TadisOn opened the pull request:
Implemented API class with a method to save stocks data to csv by given symbol

TadisOn committed:
Implemented API

Implemented API class with a method to save stocks data to csv by given symbol

ItsZil commented on Utilities/ApiHelper.cs:
```diff
@@ -0,0 +1,34 @@
+
+namespace investiciju_portfolio.Utilities
+{
+    public class ApiHelper
```
Gal galime pervadinti i StockAPI ar kazkaip panasiai?

ItsZil replied:
Reiktu pakeisti i statine klase, kad ateities funkcijas galetume naudoti paprastai

ItsZil commented on Utilities/ApiHelper.cs:
```diff
@@ -0,0 +1,34 @@
+            this._apiKey = apiKey;
+        }
+
+        public void SaveCSVFromURL(string symbol)
```
Reiktu pakeisti kad sitas grazintu double (kaina), arba atskiroje funkcijoje

ItsZil reviewed (changes requested):
dar sukurk viena funkcija kuri grazins tam tikros dienos kaina (nurodyta per parametra)

TadisOn committed:
Updated StockAPI

-Sukurtas metodas kuris gražina kainą. Parametrai : stockSymbol, daysPassed

- ApiHelper klasė pervadinta į StockAPI

TadisOn committed:
Updated StockAPI #2

-Added comments
-Renamed class
-Added getPrice() method

ItsZil committed:
Merge branch 'master' into IP-38-implement-api-that-fetches-stock-prices
